master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -178,33 +178,34 @@ PING google.com (216.58.219.206): 56 data bytes
 ### Prevent outgoing connections from pods
 
 Kubernetes NetworkPolicy does not provide a way to prevent outgoing connections from pods.  However,
-Calico does. In this section we'll create a Policy using `calicoctl` which prevents all outgoing
-connections from Kubernetes pods in the advanced-policy-demo Namespace.
+Calico does. In this section we'll create egress Policies using `calicoctl`
+which allow the outgoing connections in the advanced-policy-demo Namespace we
+want, all other egress traffic will be denied.
 
-To do this, we'll need to create a Policy which selects all pods in the Namespace, and denies
-traffic that doesn't match another Pod in the Namespace.
+To do this, we'll need to create a Policy which selects all pods in the Namespace, and allows
+egress traffic to other Pods in the Namespace.
 
```

Comment:
As noted previously, these are not proper nouns and should be lowercased: "Policy" --> "policy"; "Pods" --> "pods"; "Namespace" --> "namespace"